learn/input-variables/waypoint.hcl

```diff
@@ -0,0 +1,34 @@
+project = "example/go"
```

Comment:
Oh shoot, actually I do have some feedback! Can we change this to `example-inputvars-go`?